packages/ds-healthcare-gov/src/components/Logo/Logo.tsx

```diff
@@ -1,18 +1,19 @@
 import LogoEnSvg from './LogoEnSvg';
 import LogoEsSvg from './LogoEsSvg';
-import PropTypes from 'prop-types';
 import React from 'react';
 
-export function Logo(props) {
+type Language = 'en' | 'es';
+
+export interface LogoProps {
+  locale: Language;
```

Comment:
Doh! That wasn't my intention

Reply:
Oh, actually, it was. Because we provide the default value, it's safe in TypeScript to make this required as a prop. I guess the only case it wouldn't cover is if someone explicitly set it to null or undefined. There's no reason not to support that scenario, though, because our code supports it. I'm changing it back to optional.